classes/module/Module.php

```diff
@@ -579,7 +589,7 @@ public static function upgradeModuleVersion($name, $version)
      * Check if a module need to be upgraded.
      * This method modify the module_cache adding an upgrade list file.
      *
-     * @param $module
+     * @param Module $module
      *
      * @return bool
```

Comment:
As you're updating the phpdoc on this method, we see it can return `null` too.

Reply:
Fixed, I don't change return type for methods who return a `Db::executeS()` because it's too much return type ^^
https://github.com/PrestaShop/PrestaShop/blob/aec358d3356441b45ad2a076b9aa3923d6b8fa28/classes/module/Module.php#L1708